sonar-plugin-api/src/main/java/org/sonar/api/measures/Measure.java

```diff
@@ -43,7 +41,9 @@
 
   /**
    * Default precision when saving a float type metric
+   * @deprecated in 5.3. Decimal scale is provided by metric, not by measure.
    */
+  @Deprecated
   public static final int DEFAULT_PRECISION = 1;
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)